### PR TITLE
Add BMC Logo

### PR DESCRIPTION
**Pull request by xmulligan**

### CODE REVIEW

**Comment by xmulligan** [comment]
@shahed-al I found where to put the link

**Comment by edwarnicke** [comment]
@xmulligan Is this ready to be merged then?

**Comment by xmulligan** [comment]
@edwarnicke Yes I think it is ready

**Comment by shahed-al** [comment]
Yes it’s ready

File: layouts/shortcodes/logos.html [hunk 1]
```diff
@@ -2,6 +2,9 @@
   <div class="mt-4 col col-md-4 col-6">
     <a href="https://akamai.com" target="_blank" rel="noopener"><img src="/img/logos/akamai.png" title="Akamai" /></a>
   </div>
+  <div class="mt-4 col col-md-4 col-6">
+    <a href="https://www.bmc.com/" target="_blank" rel="noopener"><img src="/img/logos/static/BMC.png" title="BMC" /></a>
```
Comment (edwarnicke) [inline layouts/shortcodes/logos.html:6]: The link here is incorrect it should be 

```suggestion
<a href="https://www.bmc.com/" target="_blank" rel="noopener"><img src="/img/logos/BMC.png" title="BMC" /></a>
```

**Comment by edwarnicke** [comment]
@xmulligan We have a bit of a mechanical problem (see https://deploy-preview-57--inclusivenaming.netlify.app/ )... img src was wrong.  I've suggested what I think is the right thing.  If so, please correct and we can proceed to merge :)

**Comment by xmulligan** [comment]
Thanks @edwarnicke it works now. appreciate the help

**Comment by edwarnicke** [comment]
@xmulligan We all get by with a little help from our friends... I've made many many many such small errors in my day, and had most of them caught by friends in code review :)  Just paying it forward here :)

**Pull request merged**